ConfidentialClientTokenCache/Program.cs

```diff
@@ -103,6 +103,12 @@ private static IMsalTokenCacheProvider CreateTokenCache(CacheImplementationDemo
                         options.ConnectionString = @"Data Source=(localdb)\MSSQLLocalDB;Initial Catalog=TestCache;Integrated Security=True;Connect Timeout=30;Encrypt=False;TrustServerCertificate=False;ApplicationIntent=ReadWrite;MultiSubnetFailover=False";
                         options.SchemaName = "dbo";
                         options.TableName = "TestCache";
+
+                        // You don't want the SQL token cache to be purged before the access token has expired. Usually
+                        // access tokens expire after 1 hours (but this can be changed by token lifetime policies), whereas
```

Comment:
```suggestion
                        // access tokens expire after 1 hour (but this can be changed by token lifetime policies), whereas
```

Reply:
is 90 min random or for a specific reason?

Reply:
It's random (more than 60 mins)